frontend/src/layout/NavMain.tsx

```diff
@@ -46,7 +46,7 @@ export const NavMain: React.FC<Props> = ({ title, breadcrumbs, children, ...navP
         },
     }}>
         <div css={{ gridArea: "breadcrumbs" }}>{breadcrumbs}</div>
-        <h1 css={{ gridArea: "title", margin: "12px 0" }}>{title}</h1>
+        {title !== undefined && <h1 css={{ gridArea: "title", margin: "12px 0" }}>{title}</h1>}
```

Comment:
```suggestion
        {title && <h1 css={{ gridArea: "title", margin: "12px 0" }}>{title}</h1>}
```

would be shorter and would also catch `''`, or do we want an empty header for an empty title? :thinking:

Reply:
That's what I wrote first, but got this:

```
error  Unexpected nullable string value in conditional. Please handle the nullish/empty cases explicitly  @typescript-eslint/strict-boolean-expressions
```

So... disable that lint?

Reply:
Hmm, good point. No, I would leave it, that's actually useful sometimes. (I ran into `0` being falsy quite a few times already ...)

I'm still disappointed that there is no `x` in JS that completes this analogy: `??` : `||` :: `x` : `&&`. :disappointed: